tests/models/swinv2/test_modeling_swinv2.py

```diff
@@ -380,6 +436,10 @@ def test_model_from_pretrained(self):
             model = Swinv2Model.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @unittest.skip(reason="Swinv2 does not support feedforward chunking yet")
+    def test_feed_forward_chunking(self):
+        pass
```

Comment:
Why skip now?

Reply:
Since feedforward chunking is not supported by this model.